packages/block-library/src/post-comment-author/edit.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * WordPress dependencies
+ */
+import ServerSideRender from '@wordpress/server-side-render';
+
+// TODO: JSDOC types
```

Comment:
Let's either add those before we merge, or remove the comment before merging, since it doesn't make sense to have a lot of these comments in code in the repo -- we kinda know that we should add them gradually :slightly_smiling_face:

Reply:
Removed it. BTW, are Edit components should have JSDoc? Couldn't find any block Edit component with JSDoc.

Reply:
They might not currently. OTOH, my main motivation would be the typings, and those aren't very widespread in Gutenberg yet anyway.